### PR TITLE
Tao viewer store

### PR DESCRIPTION
**Pull request by SpliiT**

### CODE REVIEW

File: stores/viewer.js [hunk 1]
```diff
@@ -85,25 +85,29 @@ export const use_viewer_store = defineStore("viewer", {
...
-          // Now that the client is ready let's setup the server for us
-          viewer_call({
-            schema: schemas.opengeodeweb_viewer.create_visualization,
+            // Now that the client is ready let's setup the server for us
+            viewer_call({
+              schema: schemas.opengeodeweb_viewer.create_visualization,
+            })
+            viewer_call({
+              schema: schemas.opengeodeweb_viewer.reset,
+            })
```
Comment (BotellaA) [inline stores/viewer.js:102]: should we await for these calls? @JulienChampagnol

**Review by BotellaA — APPROVED** [review]

File: stores/viewer.js (hunk 1 above)
Comment (JulienChampagnol) [reply stores/viewer.js:102]: I would say yes, we don't want to resolve before they ended

**Review by BotellaA — APPROVED** [review]

**Review by JulienChampagnol — APPROVED** [review]

**Pull request merged**